extensions/sql-bindings/src/services/azureFunctionsService.ts

```diff
@@ -147,12 +159,21 @@ export async function createAzureFunction(connectionString: string, schema: stri
 				.withAdditionalProperties(propertyBag)
 				.withConnectionInfo(connectionInfo).send();
 		} catch (e) {
+			let errorType = utils.getErrorType(e);
 			propertyBag.quickPickStep = quickPickStep;
-			exitReason = 'error';
-			void vscode.window.showErrorMessage(utils.getErrorMessage(e));
 
-			TelemetryReporter.createErrorEvent(TelemetryViews.CreateAzureFunctionWithSqlBinding, TelemetryActions.exitCreateAzureFunctionQuickpick, undefined, utils.getErrorType(e))
+			if (errorType === 'TimeoutError') {
+				// this error can be cause by many different scenarios including timeout or error occurred during createFunction
+				exitReason = 'timeout';
+				console.log('Error creating azure function project');
```

Comment:
Same as above,

`Timed out waiting for Azure Function project to be created. This may not necessarily be an error, for example if the user canceled out of the create flow.`